packages/datetime/src/dateInput.tsx

```diff
@@ -167,16 +172,6 @@ export class DateInput extends AbstractComponent<IDateInputProps, IDateInputStat
             "pt-intent-danger": !(this.isMomentValidAndInRange(date) || isMomentNull(date) || dateString === ""),
```

Comment:
Yeah I spotted that. I think there are a few other places in the codebase too. Can circle back.